### DISCUSSION TITLE
Leftovers from uninstalled plugins pollute rtp and there is no convenient way to delete them?

### DISCUSSION THREAD

**Discussion by Anrock** [lumen-oss/rocks.nvim#304] [Q&A] [answered]
In my `.local/share/nvim/rocks/share/lua/5.1` there are folders from plugins which I don't have installed (they are not in rocks.toml). 

For example nvim-web-devicons, which was pulled as a dependency for some other plugin I actually use or used before and then uninstalled.

I guess I kinda don't care about having unused stuff on my ssd however this folder is symlinked in rocks_rtp and as such can be `require`d by other plugins and alter their behaviour. 

Specific example I've encountered is `feline` and `nvim-web-devicons`: `feline` does `pcall(require, 'nvim-web-devicons')` to check if icons are available and adjusts theme accordingly. 

Which was very unexpected for me and made me go dig around how plugin which I don't have installed still affects my config. And then I found the issue - folder with `nvim-web-devicons` wasn't deleted and thus still present in rtp.

Checking which packages are unused and cleaning them up manually is pretty inconvenient and error prone and I haven't found corresponding `Rocks` command. Help?

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

We have an open issue for this: https://github.com/nvim-neorocks/rocks.nvim/issues/77

Currently, the workaround is to run `:Rocks sync` repeatedly, until all unused dependencies have been pruned.

**Reply by mrcjkb** [reply]
The fix PR: https://github.com/nvim-neorocks/rocks.nvim/pull/305
